src/modules/ExlMd2Html.js

```diff
@@ -61,11 +61,12 @@ async function converter(mdString) {
   createTabs(document);
   createTables(document);
   createShadeBox(document);
-
+  /* FIXME: Page breaking - docs/authoring-guide-exl/using/markdown/syntax-style-guide
```

Comment:
@ahmed-musallam This is breaking for a page due to unclean HTML (div inside p/span tags). https://experienceleague.corp.adobe.com/docs/authoring-guide-exl/using/markdown/syntax-style-guide.html

Just to unblock QA activity for the time-being, we commented it out. We will revisit this once we have most of our blocks done.